packages/styles/src/mixins/_utilities.scss

```diff
@@ -100,10 +101,34 @@
     // In case rules need to be slightly adjusted
     @content;
   }
+
+  // When a browser doesn't support :has, use focus-within as a fallback. This means that focus state is displayed on focus and not on focus-visible only (except some browsers like Safari).
+  @supports not selector(:has(:focus-visible)) {
+    &:is(:focus-visible, :focus-within, .pretend-focus)#{$vendor-prefix} {
+      outline-offset: spacing.$size-line;
+      outline: spacing.$size-line solid var(--post-focus-color);
+      border-radius: $border-radius;
+
+      @include high-contrast-mode() {
+        outline-color: Highlight;
+      }
+
+      // In case rules need to be slightly adjusted
+      @content;
```

Comment:
This is a duplicated code, but I don't know how to improve that situation. We cannot use a placeholder here and the `@content` restrict the usage of another @mixin